backend/app/adapter/db/.env

```diff
@@ -1,7 +1,7 @@
 DB_HOST=localhost
-DB_PORT=5432
+DB_PORT=5433
 DB_USER=postgres
-DB_PASSWORD=password
-DB_NAME=short-test
+DB_PASSWORD=123456
+DB_NAME=tom
 
 MIGRATION_ROOT=./migration
```

Comment:
Please revert changes to this file. This is a template used for integration testing.